FIXME: remove for cygwin-1.7 !

--- origsrc/Falcon-0.8.14/modules/feathers/socket/CMakeLists.txt	2008-11-29 13:33:20.000000000 -0600
+++ src/Falcon-0.8.14/modules/feathers/socket/CMakeLists.txt	2009-01-26 00:54:33.984875000 -0600
@@ -17,6 +17,11 @@ IF(UNIX OR MAC)
       SET(SYS_SPECIFIC_LIBS /lib/libsocket.so)
    ENDIF( "${CMAKE_SYSTEM_NAME}" STREQUAL "SunOS" )
 
+   # On Cygwin 1.5, we need getaddrinfo
+   IF(CYGWIN)
+      SET(SYS_SPECIFIC_LIBS getaddrinfo)
+   ENDIF(CYGWIN)
+
 ELSE(UNIX OR MAC)
    #We must configure socket_sys_win.cpp.in with the correct location of SDK
    IF( "${CMAKE_GENERATOR}" STREQUAL "MinGW Makefiles" )
--- origsrc/Falcon-0.8.14/modules/feathers/socket/socket_sys_unix.cpp	2008-10-14 16:31:01.000000000 -0500
+++ src/Falcon-0.8.14/modules/feathers/socket/socket_sys_unix.cpp	2009-01-26 00:53:50.938000000 -0600
@@ -31,6 +31,11 @@
 #include <string.h>
 #include "socket_sys.h"
 
+#ifdef __CYGWIN__
+#include <dummyin6.h>
+#include <getaddrinfo.h>
+#endif
+
 // Sun doesn't provide strerror_r
 #if ( defined (__SVR4) && defined (__sun) )
 static int strerror_r(int errnum, char * buf, unsigned n)
