--- origsrc/Falcon-0.9.4.4/core/engine/core_module/math_ext.cpp	2009-08-08 05:59:47.000000000 -0500
+++ src/Falcon-0.9.4.4/core/engine/core_module/math_ext.cpp	2009-12-29 22:29:22.014666200 -0600
@@ -502,7 +502,7 @@ FALCON_FUNC  flc_round ( ::Falcon::VMach
    else if ( num->type() == FLC_ITEM_NUM )
    {
       // Or windows or solaris, use a simple round trick.
-      #if defined(_MSC_VER) || ( defined (__SVR4) && defined (__sun) )
+      #if defined(_MSC_VER) || defined(__CYGWIN__) || ( defined (__SVR4) && defined (__sun) )
          numeric n = num->asNumeric();
          numeric intpart;
          numeric fractpart = modf(n, &intpart );
--- origsrc/Falcon-0.9.4.4/core/include/falcon/dll_dl.h	2009-06-19 06:01:44.000000000 -0500
+++ src/Falcon-0.9.4.4/core/include/falcon/dll_dl.h	2009-12-29 22:29:52.566413600 -0600
@@ -44,7 +44,11 @@ public:
 
    DllFunc getSymbol( const String &sym_name ) const ;
    static bool isDllMark( char ch1, char ch2 );
+#ifdef __CYGWIN__
+   static const char *dllExt() { return "dll"; };
+#else
    static const char *dllExt() { return "so"; };
+#endif
 };
 
 typedef DllLoader_dl DllLoader;
